### DISCUSSION TITLE
Prevent scrolling to top on reload

### DISCUSSION THREAD

**Discussion by ArianYN** [sqlpage/SQLPage#484] [Q&A] [answered]
Is there a way to stop the page from scrolling to the top, when changing something in the url.
For example, if I have a parameter in the URL: ?id=1
When this id changes, from a button the page refreshes and scrolls back to the top.

Is it possible to prevent this?

**Comment by lovasoa** [comment, marked as answer]
Hello ! You can make a link to a particular point in the page using a [URL fragment](https://en.wikipedia.org/wiki/URI_fragment). For instance:

```sql
select 
    'button' as component;

select 
    '?id=1#mybutton'  as link,
    'My Button' as title,
    'mybutton' as id;

```

**Reply by ArianYN** [reply]
Oh. So the link, will scroll down to specified id in the URL?

Does this work with all components?

**Reply by lovasoa** [reply]
This works with all components that have an `id` property, that is most of them. The list of available properties is available in the documentation.

**Reply by ArianYN** [reply]
Okay.

But there is not a way for it to just stay. (So it doesn't go to the top first, and then scrolls), but just stays at one point when refreshing?
